src/Timeline.js

```diff
@@ -192,8 +188,8 @@ export default class Timeline extends Axis {
       : this._d3Scale.range();
 
     if (this._buttonBehaviorCurrent === "ticks") {
-      domain[0] = dateParser(closest(domain[0], ticks));
-      domain[1] = dateParser(closest(domain[1], ticks));
+      domain[0] = date(closest(domain[0], ticks));
+      domain[1] = date(closest(domain[1], ticks));
     }
```

Comment:
shouldn't all the `date` instances in `_updateDomain` also be `this._dateParser`?

Reply:
@davelandry in my user-case, `this._dateParser` overrides the date behavior inside `render`, if I put the function in `_updateDomain`, I'm doing twice the conversion of the date

Reply:
I don't follow... this logic (using the `date` function) takes the result of `closest(domain[0], ticks)` and turns it into a JavaScript Date object. Is the return of `closest(domain[0], ticks)` not one of the user-defined dates?

Reply:
@davelandry because in` _updateDomain` we are receiving the ticks processed previously by `date` or `this._dateParser`. Using a second time `this._dateParser` break the timeline.